### PR TITLE
Fix annoying typography variant deprecated warning.

### PR DESCRIPTION
**Pull request by sri2k1us**
See https://material-ui.com/style/typography/#migration-to-typography-v2

P.S: I may have missed few spots. If so, please update them in your next PR. 🙏

### CODE REVIEW

File: react-components/src/lib.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ const getDefaultTheme = () => createMuiTheme({
     typography: {
         button: {
             textTransform: "none",
-        }
+        },
+        useNextVariants: true,
```
Comment (psarando) [inline react-components/src/lib.js]: I'm wondering if we should hold off on these updates.

By just setting `useNextVariants: true`, the delete icons in the `EditMetadataTemplate` expansion panel headers go from looking like this
screen shot 2019-02-27 at 12 54 10 pm
to looking like this
screen shot 2019-03-05 at 4 52 06 pm
This happens in both Chrome and Firefox, and it doesn't matter which `Typography` variants are used in `EditMetadataTemplate`.

Comment (sri2k1us) [reply react-components/src/lib.js]: Weird. Looks Ok for me...
delete-btn

Google chrome - Version 72.0.3626.121 (Official Build) (64-bit)

Comment (sri2k1us) [reply react-components/src/lib.js]: It does look like your screenshot in storybook 😕

Comment (sri2k1us) [reply react-components/src/lib.js]: delete-with-margin


Fixed it with margin. Looks good in Belphe as well.

Comment (psarando) [reply react-components/src/lib.js]: I think the reason it looked OK in your 1st screenshot is there is only an attribute name header, but no attribute description subheading.

Adding the margin appears to fix it, though.

Nice find! 👍

**Comment by sri2k1us** [comment]
Thanks for reviews.

**Pull request merged**